hoomd/md/update.py

```diff
@@ -213,3 +214,95 @@ def _attach(self):
     def summed_exchanged_momentum(self):
         R"""Returned the summed up exchanged velocity of the full simulation."""
         return self._cpp_obj.summed_exchanged_momentum
+
+
+class ActiveRotationalDiffusion(Updater):
+    r"""Updater to introduce rotational diffusion with an active force.
+
+    This updater works directly with an `hoomd.md.force.Active` or
+    `hoomd.md.force.ActiveOnManifold` instance to update rotational diffusion
+    for simulations with active forces.
+
+
+    The diffusion of the updater  follows :math:`\delta \theta / \delta t =
+    \sqrt{2 D_r / \delta t} \Gamma`, where :math:`D_r` is the rotational
+    diffusion constant, and the gamma function is a unit-variance random
+    variable, whose components are uncorrelated in time, space, and between
+    particles. In 3D, :math:`\hat{p}_i` is a unit vector in 3D space, and
+    diffusion follows :math:`\delta \hat{p}_i / \delta t = \sqrt{2 D_r /
+    \delta t} \Gamma (\hat{p}_i (\cos \theta - 1) + \hat{p}_r \sin \theta)`,
+    where :math:`\hat{p}_r` is an uncorrelated random unit vector. The
+    persistence length of an active particle's path is :math:`v_0 / D_r`.
+    The rotational diffusion is applied to the orientation vector/quaternion
+    of each particle. When used with `hoomd.md.force.ActiveOnManifold`,
+    rotational diffusion is performed tangent plane of the manifold.
+
+    Tip:
+        Use `hoomd.md.force.Active.create_diffusion_updater` to construct
+        a `ActiveRotationalDiffusion` instance.
+
+    Args:
```

Comment:
Move the Args section to the top of the docstring so that it appears close to the argument list when sphinx renders it.